numpy/core/setup.py.orig

```diff
@@ -0,0 +1,1173 @@
+import os
```

Comment:
This backup of `setup.py` should be removed.

Reply:
still